foundry/https_download.py

```diff
@@ -1,3 +1,10 @@
+"""
+https_download.py
```

Comment:
As above. Don't need to say the filename in the docstring. It will surely go out of date.